contracts/OysterPearl.sol

```diff
@@ -172,26 +179,30 @@ contract OysterPearl {
    *
    * When an address is buried; only claimAmount can be withdrawn once per epoch
    */
-  function bury() public returns (bool success) {
-    // The address must be previously unburied
-    require(!buried[msg.sender]);
-
-    // An address must have at least claimAmount to be buried
-    require(balances[msg.sender] >= claimAmount);
-
-    // Prevent addresses with large balances from getting buried
-    require(balances[msg.sender] <= retentionMax);
-
-    // Set buried state to true
-    buried[msg.sender] = true;
-
-    // Set the initial claim clock to 1
-    claimed[msg.sender] = 1;
-
-    // Execute an event reflecting the change
-    Bury(msg.sender, balances[msg.sender]);
-    return true;
-  }
+   function bury(bytes32 hash) public returns (bool success) {
+     // The hash must be previously unburied
+     require(!buried[hash]);
+        
+     // claimAmount is able to be transferred to this contract
+     require(_transfer(msg.sender, address(this), claimAmount));
+        
+     // Assign buried PRL to this hash
+     hashBalances[hash] += claimAmount;
+        
+     // Assign broker to this hash
+     buryBroker[hash] = msg.sender;
+        
+     // An address must have at least claimAmount to be buried
+     require(hashBalances[hash] >= claimAmount);
```

Comment:
Do you still need this guard? Doesn't seem possible to be false here since you add `claimAmount` above.

Reply:
Nope, not needed